src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue31680.cs

```diff
@@ -0,0 +1,29 @@
+﻿#if TEST_FAILS_ON_WINDOWS   // Scrolling the CollectionView using App.ScrollDown is not working on Windows platform.  
```

Comment:
Is not finding the CollectionView or does nothing and not scroll?

Reply:
@jsuarezruiz , It does find the CollectionView, but it doesn’t scroll when the CollectionView contains a CarouselView as its ItemTemplate. Scrolling works fine for a CollectionView without a CarouselView as the template.